Plugins/Flow.Launcher.Plugin.Program/Settings.cs

```diff
@@ -14,9 +14,13 @@ public class Settings
         public bool EnableStartMenuSource { get; set; } = true;
 
         public bool EnableRegistrySource { get; set; } = true;
+        public string CustomizedExplorer { get; set; } = Explorer;
+        public string CustomizedArgs { get; set; }
```

Comment:
All the other things are good, except I am not sure whether it would be ok to not give this one an initail value?

Reply:
Oh since I am going to change the default value, there is nothing to concern. Thank you for those changes!